src/api/database/find.rst

```diff
@@ -1141,6 +1152,14 @@ but including it makes the intent of the selector clearer and will make
 it easier to take advantage of future improvements to query planning
 (e.g. automatic selection of partial indexes).
 
+.. note::
+    An index with fields will never be used, unless the selector includes
+    all of the fields indexed. For instance, if an index contains ``["a". "b"]``
```

Comment:
From this example, I understand that
| Selector | Index | Index is used |
|----------|-------|---------------|
| a        | a, b  | no            |
| a, b     | a     | yes           |

Is that correct?